### ISSUE TITLE
Help on setting the line index

### ISSUE THREAD

**Issue by ChienDevIT** [morrisjs/morris.js#769]
Hello. I'm trying to build a car reporting system using your chart.

But I met a great barrier ;-( !
![image](https://user-images.githubusercontent.com/32790030/44672793-b924bc00-aa53-11e8-8089-44ae03fe97a9.png)

The current vehicle number is integer, and should be sequential, but in the report it is an odd number.
I tried to look on the internet but helpless. Does anyone have a good idea to help me to set it up with an array.

Thanks,

**Comment by pierresh** [comment]
Hello, please give your dataset so that we could reproduce your situation

**Comment by ChienDevIT** [comment]
It's like this
```
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/morris.js/0.5.1/morris.css">
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/morris.js/0.5.1/morris.min.js"></script>

<div id="myfirstchart" style="height: 250px;"></div>

<script type="text/javascript">
	new Morris.Line({
	element: 'myfirstchart',
	data: [
	<?php
	for($i = 0; $i <20; $i++) {
		$time = time() + rand(-99999, +99999);
		$time = $time * 1000;
		$value1 = rand(0, 13);
		$value2 = rand(46, 90);
		$value3 = rand(1, 3);
		$value4 = rand(0, 10);
		$value5 = rand(10, 20);
		echo "{ time: {$time}, value1: {$value1}, value2: {$value2}, value3: {$value3}, value4: {$value4}, value5: {$value5} },";
	}
	?>
	],
	xkey: 'time',
	ykeys: ['value1', 'value2', 'value3', 'value4', 'value5'],
	labels: ['Value', 'Value', 'Value', 'Value', 'Value'],
	parseTime: true,
});

</script>
```

I think it happens because the data is far apart.
I am wanting my vertical axis, it has a setting such as "step", to be able to create indexes 1, 2, 3, 4, ... or 5, 10, 15, 20, ...

**Comment by pierresh** [comment]
Ah ok, I think I better understand your needs. I think you need to compute the max value of your data, then assign to the options **ymax** a value that would make horizontal lines matches with a integer value. In your example, as the max value is 13 and there are 4 lines by default, you need to set the **ymax**  to 16.

**Comment by ChienDevIT** [comment]
Thanks for answer !

**Comment by pierresh** [comment]
You are welcome, just for record, the formula to define the **ymax** could be _`Math.ceil(max value in data/4) * 4`_
